code/datums/mutations/body.dm

```diff
@@ -472,16 +472,18 @@
 	power_coeff = 1
 
 /datum/mutation/human/densebones/on_acquiring(mob/living/carbon/human/owner)
-	. = ..()
-	owner?.physiology?.armor?.melee += 5
-	owner?.physiology?.armor?.wound += 10
+	if(..())
+		return
+	owner.physiology.armor.melee += 5
+	owner.physiology.armor.wound += 10
 	if(GET_MUTATION_POWER(src) > 1)
 		ADD_TRAIT(owner, TRAIT_HARDLY_WOUNDED, "genetics")
 
 /datum/mutation/human/densebones/on_losing(mob/living/carbon/human/owner)
-	. = ..()
-	owner?.physiology?.armor?.melee += 5
-	owner?.physiology?.armor?.wound += 10
+	if(..())
+		return
+	owner.physiology.armor.melee += 5
```

Comment:
Shouldn't this and the next one be -=

Reply:
spotted this issue while I was working on a fix myself.